dist/variables/ds6/color-variables.less

```diff
@@ -9,6 +9,7 @@
 @color-grey5: #767676;
 @color-grey6: #414141;
 @color-grey7: #111820;
+@color-grey8: #616161;
```

Comment:
Did design system actually add a new grey to the colour palette?

Reply:
I'm not sure; I just know I couldn't find that color here in the variables. I found that color on Figma

Reply:
Okay. The colour palette and naming needs to strictly match what we have in the design system palette. If they don't call this out as "grey 8" then we shouldn't either.